### ISSUE TITLE
Values files should be looked for relative to course.yaml

### ISSUE THREAD

**Issue by sudermanjr** [FairwindsOps/reckoner#138]
If I have a values file that is relative to the course.yaml, but I run reckoner from a different directory, then reckoner cannot find the values file.

Example:

```
└─ tree
.
├── resources
│   ├── course-demos.yaml
│   └── demo-manifests
│       ├── demo-values.yaml
│       └── load-generator-values.yaml
└── vars.yml
```

Contents of course-demos.yaml
```
repository: stable
namespace: demo
minimum_versions:
  helm: 2.14.3
  reckoner: 2.0.0
repositories:
  incubator:
    url: https://kubernetes-charts-incubator.storage.googleapis.com
  stable:
    url: https://kubernetes-charts.storage.googleapis.com
  fairwinds-stable:
    url: https://charts.fairwinds.com/stable
  fairwinds-incubator:
    url: https://charts.fairwinds.com/incubator
charts:
  load-generator:
    hooks:
      pre_install:
        - kubectl create ns load > /dev/null 2>&1 || true
    version: 0.1.1
    repository: fairwinds-incubator
    namespace: load
    files:
      - demo-manifests/load-generator-values.yaml
```

```
└─ reckoner plot resources/course-demos.yaml
2019-09-17 15:44:25  root[30860]     INFO  Installing load-generator
2019-09-17 15:44:25  root[30860]     INFO  Running pre_install hook...
2019-09-17 15:44:26  root[30860]     INFO  Ran Hook: 'kubectl create ns load > /dev/null 2>&1 || true'
2019-09-17 15:44:26  root[30860]     INFO  pre_install hook ran successfully
2019-09-17 15:44:28  root[30860]    ERROR  Helm upgrade failed on load-generator
2019-09-17 15:44:28  root[30860]    ERROR  ERROR: Chart failed to install.
2019-09-17 15:44:28  root[30860]    ERROR   - load-generator
2019-09-17 15:44:28  root[30860]    ERROR  Stopping chart installations due to an error! Some of your charts may not have been installed!
⛵🔥 Encountered errors while running the course ⛵🔥

* * * * *

Chart Name: load-generator
Status: Failed
Error Reason: Command Failed with output below:
STDOUT: Release "load-generator" does not exist. Installing it now.

STDERR: Error: open demo-manifests/load-generator-values.yaml: no such file or directory

COMMAND: helm upgrade --install load-generator fairwinds-incubator/load-generator --namespace load --version 0.1.1 -f demo-manifests/load-generator-values.yaml
```

My personal expectation would be to be able to use relative paths, however I also understand if that doesn't end up being the case.

**Comment by endzyme** [comment]
Interesting! This used to be a feature and I believe still works with files not in a folder. I'll take a look at this for sure

**Comment by endzyme** [comment]
Fixed in v2.1.0
